Project.toml

```diff
@@ -12,7 +12,9 @@ StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
 
 [compat]
+CategoricalArrays = "0.7"
 DataAPI = "1.1"
+DataFrames = "0.20, 0.21"
```

Comment:
I've checked that 0.19 doesn't work since it's incompatible with the allowed Tables versions.